Signum.Engine.Extensions/Mailing/EmailLogic.cs

```diff
@@ -351,6 +351,13 @@ public virtual void Send(EmailMessageEntity email)
 
                 try
                 {
+                    using (Transaction tr = Transaction.ForceNew())
+                    {
+                        email.State = EmailMessageState.Draft;
+                        email.Save();
+                        tr.Commit();
+                    }
+

```

Comment:
When used from AsyncEmailSender, this change is moving the Email from ReadyToSend to Draft. Is this considered?